packages/@o3r/telemetry/src/schematics/index.ts

```diff
@@ -47,7 +48,17 @@ export const createSchematicWithMetrics: SchematicWrapper =
           error
         };
         context.logger.debug(JSON.stringify(data, null, 2));
-        await sendData(data, context.logger);
+        let shouldSendData = !!(options as any).o3rMetrics;
+        if (shouldSendData && tree.exists('/nx.json')) {
```

Comment:
actually in `nx` context, the tree is populated with a `angular.json` resulting of the merge of `nx.json` and the `project.json` files.
Only the second if should be ok.
note: working only with schematics' tree (not for builder)